layouts/page/polars-gpu-engine.html

```diff
@@ -190,12 +190,12 @@ <h2 class="accordion-header">
             <div class="accordion-item">
               <h2 class="accordion-header">
                 <button class="accordion-button wrappable-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapse2" aria-expanded="false" aria-controls="collapse2">
-                  Can I use the Polars GPU engine on a Windows machine?
+                  How many GPUs does the Polars GPU Engine work with?
```

Comment:
Can you change line 182 to `pip install polars[gpu]` and remove the `--extra-index-url=https://pypi.nvidia.com`?